libs/kotaemon/kotaemon/loaders/adobe_loader.py

```diff
@@ -146,6 +146,7 @@ def load_data(
                         "page_label": page_number,
                         "file_name": filename,
                         "file_path": filepath,
+                        **(extra_info if extra_info is not None else {}),
```

Comment:
`if extract_info else {}` is good enough

Reply:
The code has been modified according to the suggestion.